edb/schema/utils.py

```diff
@@ -391,17 +452,25 @@ def find_item_suggestions(
 
 
 def enrich_schema_lookup_error(
-        error, item_name, modaliases, schema, *,
-        item_types, suggestion_limit=3, name_template=None,
-        collection=None, condition=None, context=None):
+        error: errors.EdgeDBError,
+        item_name: Union[sn.SchemaName, str, None],
```

Comment:
`sn.SchemaName` is a subclass of `str`, so I wonder if `Optional[str]` is enough here.

Reply:
Right. :+1: